Week-1/tanmay-sinha/GUIapp.py

```diff
@@ -0,0 +1,124 @@
+import tkinter as tk
```

Comment:
- Bad Whitespaces everywhere.
You can refer: https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements